src/models/Order.js

```diff
@@ -70,8 +75,8 @@ const orderSchema = new mongoose.Schema(
 const updateOrderStatus = async () => {
   try {
     // Get the current date and subtract 1 minute
-    const oneMinutesAgo = new Date(Date.now() - 1 * 60 * 1000);
-
+    //const oneMinutesAgo = new Date(Date.now() - 1 * 60 * 1000);
+    const oneMinutesAgo = new Date(Date.now() - 72000000); // for tests -delete later
```

Comment:
This is fine; should probably chat with the team to determine what's a reasonable amount of time. And we could even make it an envirionment variable so that we can more easily have a longer time for local dev vs in production or vice vers.

Reply:
@akosasante thank you! will commit it -please take a look if it's ok